<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alt::AgreementWizardPage</class>
 <widget class="QWizardPage" name="alt::AgreementWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="title">
   <string>License Agreement</string>
  </property>
  <property name="subTitle">
   <string>Read the license agreement. If you accept the terms of the agreement, select &quot;Yes, I agree to the terms&quot; and press &quot;Finish&quot;.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="licenseTextBrowser"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="agreementCheckBox">
       <property name="text">
        <string>Yes, I agree to the terms</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
